### PR TITLE
Remove gauge

### PR DESCRIPTION
**Pull request by liyuqian**
It's now replaced by https://github.com/flutter/engine/pull/18752

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,6 @@ These are the available packages in this repository.
 |--------|-----|
 | [animations](./packages/animations/) | [![pub package](https://img.shields.io/pub/v/animations.svg)](https://pub.dev/packages/animations) |
 | [fuchsia_ctl](./packages/fuchsia_ctl/) | [![pub package](https://img.shields.io/pub/v/fuchsia_ctl.svg)](https://pub.dev/packages/fuchsia_ctl) |
-| [gauge](./packages/gauge/) | [![pub package](https://img.shields.io/pub/v/gauge.svg)](https://pub.dev/packages/gauge) |
```
Comment (goderbauer) [inline README.md:38]: We should put something on pub to say that this is no longer maintained or delete it from there or whatever the style is for deprecated packages.

**Review by goderbauer — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (liyuqian) [reply README.md:38]: Yes, I marked it as **discontinued** on pub.dev.

Comment (creativecreatorormaybenot) [reply README.md:38]: You could link to this PR as well or explain the replacement by engine 👍

**Pull request merged**